docs/decisions/0003-remove-warehouse-database.md

```diff
@@ -0,0 +1,44 @@
+# 3. Remove warehouse database
+
+Date: 2019-10-16
+
+## Status
+
+Accepted
+
+## Context
+
+The data warehouse was intended to house data for all CommCareHQ reports.
+The warehouse would replace Elasticsearch in almost all contexts that it is currently used.
+The migration began in 2017 with the Application Status report and the effort
+to move the report to the warehouse and ensure it is stable, performs well and
+provides the same features as the ES-backed reports was much higher than
+anticipated.
+
+## Decision
+
+To reduce our infrastructure dependencies and focus our efforts on existing databases,
+we have decided to remove the warehouse and stop any efforts to iterate on it.
+
+This decision is not because we believe that the warehouse is a worse implementation than Elasticsearch.
+This decision is because we believe that with our current priorities, we will
+not be able to spend the appropriate amount of time to make the warehouse a
+robust solution for generic reports in the near future.
+Because no current reports are backed by the warehouse, it is an important time
```

Comment:
There is at least one [ICDS SMS feature](https://github.com/dimagi/commcare-hq/pull/21581) that depends on the warehouse. Not sure if that is relevant to this decision.

Reply:
Sravan started changing that to use a dashboard table two weeks ago https://github.com/dimagi/commcare-hq/pull/25550 (I don't actually know the motivation behind it, it was separate from any of this)